### PR TITLE
fix(workloads): export the workloads API via the newrelic package

### PR DESCRIPTION
**Pull request by ctrombley**

### CODE REVIEW

File: pkg/workloads/workload.go [hunk 1]
```diff
@@ -244,7 +244,7 @@ const (
 		` } }`
 
 	duplicateWorkloadMutation = `
-		mutation($accountId: Int!, $sourceGuid: EntityGuid!, $workload: WorkloadDuplicateInput!) {
+		mutation($accountId: Int!, $sourceGuid: EntityGuid!, $workload: WorkloadDuplicateInput) {
```
Comment (zlesnr) [inline pkg/workloads/workload.go]: Does the `!` here just mean the field is required?

File: pkg/workloads/workloads.go [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// Package workloads provides a programmatic API for interacting with New Relic
```
Comment (zlesnr) [inline pkg/workloads/workloads.go]: Yum, package docs.

Comment (zlesnr) [reply pkg/workloads/workloads.go]: Oh I see there is also a `doc.go` above.  Are both points of documentation requried?

File: pkg/workloads/workload.go (hunk 1 above)
Comment (sanderblue) [reply pkg/workloads/workload.go]: Means "non-nullable", so basically yeah

File: pkg/workloads/workloads.go (hunk 2 above)
Comment (ctrombley) [reply pkg/workloads/workloads.go]: No, just one will do... I'll clean this up in a future PR

**Pull request merged**